core/src/main/java/com/datastax/oss/driver/internal/core/metadata/schema/parsing/TableParser.java

```diff
@@ -226,6 +226,50 @@ TableMetadata parseTable(
         indexesBuilder.build());
   }
 
+  TableMetadata parseVirtualTable(
```

Comment:
👍 for exposing this as a separate method, that leads to a bit of duplication but it's easier to follow.